svcomp-benchmark/results/README.md

```diff
@@ -0,0 +1,5 @@
+Data collected from running [RV-Match](https://runtimeverification.com/match) on the [SV-COMP Benchmark Verification Tasks](https://github.com/sosy-lab/sv-benchmarks). These results were collected on April 13th, 2016. More details about the verification tasks, and the structure of the benchmark can be found at [SV-COMP's website](http://sv-comp.sosy-lab.org/2016/index.php).
```

Comment:
@msaxena2 I think what you need is a top-level README explaining all the above and giving people the big picture, and from there refer people to the two READMEs in the two folders, results and scripts.